Guide_1_Send_Payment.md

```diff
@@ -1,31 +1,112 @@
 # Guide: Making a Payment
 
-_(Note: Are there any specific steps required for different payment schemes (SWIFT etc.)? Do the steps in this guide work the same for any scheme? Should be mentioned somehow)_
+In this guide you will learn how to use the Form3 Payments API to make a payment through Faster Payments Service (FPS). The guide covers the following steps:
+- Sending the payment
+- Tracking the payment progress manually
+- Tracking the payment progress automatically using subscriptions
 
-In this guide you will learn how to use the Form3 Payments API to make a payment. The guide covers the following steps:
-- Send the payment
-- Track the payment progress manually
-- Track the payment progress automatically using subscriptions
+Each step is illustrated with Python code snippets, so you can execute each step as you read along. The snippets are ready-to-run programs and work in Python's interactive console.
 
 ## Prerequisites:
 Before you start, make sure you have the following things ready to go:
-- API credentials. Contact your Form3 contact to obtain your credentials. _(Note: Better name for this role? Who do they have to contact?)_
-- Register your bank ID and BIC with Form3
-- Sign up for Form3's scheme simulator. This is a sandbox environment that you can use to simulate transactions in order to test your application. _(Note: in the API this is called "payment simulator". Which term should we use?)_
+- API credentials. Contact Form3 to obtain your credentials.
+- Register your UK sortcodes and BICs with Form3.
+- Sign up for Form3's scheme simulator for FPS payments. This is a sandbox environment that you can use to simulate transactions in order to test your application.
+- If you want to run the Python code snippets, make sure you have [Python 2.7](https://www.python.org/downloads/) installed. You also need the requests package. The easiest way is to install it through [Pip](https://docs.python.org/2.7/installing/index.html): `pip install requests`
 
 
 ## Get a Bearer Token
 
-As a first step, you need to obtain a bearer token using your API credentials. This token will be used to authenticate yourself to the Form3 server.
+```python
+import requests
 
-_(Note: Should we elaborate on how to do that?)_
+### Replace these variables with your own data! ###
+client_id = 'YOUR CLIENT ID HERE'
+client_secret = 'YOUR CLIENT SECRET HERE'
+
+auth_payload = "grant_type=client_credentials"
+auth_url = 'https://api.tabla.env.form3.tech/v1/oauth2/token'
```

Comment:
we should change this to 'api.test.form3.tech'

Reply:
Changed the URL in all snippets and code files.